lib/systems/inspect.nix

```
@@ -82,7 +82,10 @@ rec {
     isMusl         = with abis; map (a: { abi = a; }) [ musl musleabi musleabihf muslabin32 muslabi64 ];
     isUClibc       = with abis; map (a: { abi = a; }) [ uclibc uclibceabi uclibceabihf ];
 
-    isEfi = [
+    isUefi         = { /* kernel = kernels.windows; replace with none? */ abi = abis.coff; vendor = vendors.pc; }; # this may not be restrictive enough? TODO
```

```suggestion
    isUefi         = { abi = abis.uefi; kernel = kernels.none; };
```

The most distinctive thing about UEFI, from a "linker's point of view" is that it has its own calling convention (ABI).  Indeed, when Linux prostrates itself to venerate the firmware, it assumes the [`EfiApi` calling convention](https://wiki.osdev.org/Using_UEFI_Runtime_Services_in_your_Kernel).